.github/workflows/build-and-publish-TestPyPI.yml

```diff
@@ -48,8 +50,109 @@ jobs:
     - name: Publish Distribution to TestPyPI
       uses: pypa/gh-action-pypi-publish@v1.9.0
       with:
+        verbose: true
         password: ${{ secrets.TEST_PYPI_API_TOKEN }}
         repository_url: https://test.pypi.org/legacy/
+    - name: Handling Failure Delete tag
```

Comment:
could you add a comment for this job and explain what it is supposed to do briefly?

Reply:
[85e44b3](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/435/commits/85e44b3340e6af4c1d514896649dc5a1cc9a8372)

added comment.

"Handling Failure Delete tag" deletes the tag created by the bumpversion action when the job exits due to failure.